hushline/routes.py

```diff
@@ -95,26 +122,17 @@ def inbox() -> Response | str:
             flash("👉 Please log in to access your inbox.")
             return redirect(url_for("login"))
 
-        messages = db.session.scalars(
-            db.select(Message).filter_by(user_id=user.id).order_by(Message.id.desc())
-        ).all()
-        secondary_users_dict = {su.id: su for su in user.secondary_usernames}
-
         return render_template(
             "inbox.html",
             user=user,
-            messages=messages,
-            secondary_usernames=secondary_users_dict,
             is_personal_server=app.config["IS_PERSONAL_SERVER"],
         )
 
     @app.route("/to/<username>", methods=["GET"])
     def profile(username: str) -> Response | str:
         form = MessageForm()
-        user = db.session.scalars(
-            db.select(User).filter_by(primary_username=username).limit(1)
-        ).first()
-        if not user:
+        uname = Username.query.filter_by(_username=username).one_or_none()
```

Comment:
Can we use the session API? This style of query is deprecated.

Reply:
This isn't deprecated in `Flask-SqlAlchemy` unless I missed something. Was it deprecated upstream in SqlAlchemy itself?

Reply:
Yeah, in sqlalchemy. We were originally suppressing warnings when running tests but that included a bunch of valid ones. The session API is more consistent with the actual SQL so we made the switch.

Reply:
Where are these being suppressed? And is there a ticket for this already? I don't see a config that mentions deprecations or warnings

Reply:
Previously, we suppressed warnings when running tests, https://github.com/scidsg/hushline/blame/7f33029c4f0cfb90ede431b20109b778a09fa8eb/Makefile#L45C10-L45C44. I don't know why this decision was made originally, but I assume it's because of the large number of deprecation warnings from sqlalchemy.

Reply:
I must have removed that in my pytest-cov PR (oops), but I don't get any deprecation warnings other than one for a ticket I opened.

I don't think this query style is deprecated, and I think it's preferable for legibility. Less to type, and shorter lines so that they don't get broke up weirdly by the formatter. Plus, I think (bikeshedding lol) `User.query.all()` is more readable than `db.session.scalars(db.session.select(User).all())` since it removes all the boilerplate calls.

Reply:
Flask-sqlalchemy notes that the query interface is considered legacy and that the session API is preferred, https://flask-sqlalchemy.palletsprojects.com/en/3.1.x/legacy-query/.

Reply:
I'll fix it, but I would like to just say the upstream devs are wrong and those "legacy" interfaces are so much fucking better lmfao